cx/slack/test/server_test.js

```diff
@@ -23,28 +23,109 @@ describe('slackToDetectIntent()', () => {
     });
 });
 
-describe('detectIntentToSlackMessage()', () => {
+describe('convertToSlackMessage()', () => {
     channel_id = 54321
 
-    const slackRequest = {
+    const slackTextRequest = [{
         channel: channel_id,
-        text: 'Test Response Text\n'
-    };
+        text: 'Test Response Text'
+    }];
 
-    const dialogflowResponse = {
+    const dialogflowTextResponse = {
         queryResult: {
-            text: 'Test Text',
-            languageCode: 'en',
             responseMessages: [{
                 text: {
-                    text: 'Test Response Text'
+                    text: ['Test Response Text']
                 }
             }]
         }
     };
 
-    it('should convert detectIntent response to a Slack text message request.', async function () {
-        assert.deepStrictEqual(detectIntentToSlackMessage(
-            dialogflowResponse, channel_id), slackRequest)
+    const slackImageRequest = [{
+        channel: channel_id,
+        blocks: [{
+            type: "image",
+            image_url: "http://example.com/image",
+            alt_text: "Example image."
+        }]
+    }];
+
+    const dialogflowImageResponse = {
+        queryResult: {
+            responseMessages: [{    
+                payload: {
+                    fields: {    
+                        "blocks":{"listValue":{"values":[{"structValue":{"fields":{
+                            "type":{"stringValue":"image","kind":"stringValue"},
+                            "image_url":{"stringValue":"http://example.com/image",
+                            "kind":"stringValue"},
+                            "alt_text":{"stringValue":"Example image.","kind":"stringValue"}}},
+                        "kind":"structValue"}]},"kind":"listValue"}
```

Comment:
nit: maybe format the message so it's easier to read.

Reply:
I have restructured the code to be more readable but because it is protobuf it still looks a little cluttered.

Reply:
Looks like this one for image was not formatted (although the one of button was already formatted.)

Reply:
Fixed it.